Controllers/InputModels/TodoitemInputModel.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using Taskmanager.Data.Entities;
+
+namespace Taskmanager.Controllers.InputModels
+{
+    public class TodoitemInputModel
+    {
+        [Required]
+        [StringLength(300, MinimumLength = 1, ErrorMessage = "Todoitem must be between 1 and 3 characters")]
+        public string Title { get; set; }
+        [RegularExpression(@"^[0-9]{0,1}$", ErrorMessage = "IsFinished can have 0 or 1 value")] 
+        public long Isfinished { get; set; }
```

Comment:
Why not just have boolean in InputModel? Even if db doesn't support bool it should not affect your API